lib/facilities/ppms/client.rb

```diff
@@ -15,8 +15,16 @@ class Client < Common::Client::Base
 
       # https://dev.dws.ppms.va.gov/swagger/ui/index#!/GlobalFunctions/GlobalFunctions_ProviderLocator
       def provider_locator(params)
-        qparams = build_params(params)
-        response = perform(:get, 'v1.0/ProviderLocator?', qparams)
+        qparams = provider_locator_params(params)
+        response = perform(:get, 'v1.0/ProviderLocator', qparams)
+        return [] if response.body.nil?
+
+        Facilities::PPMS::Response.from_provider_locator(response, params)
+      end
+
+      def pos_locator(params, pos_code)
+        qparams = pos_locator_params(params, pos_code)
```

Comment:
qparams == query params?

Reply:
it's the request params that we are sending to ppms. I left it the way it was done originally.